my-webxr-app/src/data/DataLayer.tsx

```diff
@@ -17,9 +17,9 @@ import DataPoint from '../repository/DataPoint';
  * The Data Layer provides a set of methods for working with CSV and PCA data.
  */
 export default class DataLayer implements DataAbstractor {
-  private repository: Repository;
+  protected repository: Repository;
 
-  private isFirstBatch: boolean;
+  protected isFirstBatch: boolean;
```

Comment:
minor: why do we have to change access modifier for isFirstBatch?

Reply:
I changed it to be consistent with the above repository variable. I'm not sure we'd ever need to expose this variable, but 🤷‍♂️

Reply:
we might if we want to reset the flag every time user load another file/ URL. But that is out of scope for now